test/dataframerow.jl

```diff
@@ -248,4 +243,39 @@ module TestDataFrameRow
         @test df[1, 1] == 200
         @test_throws KeyError dfr2.x2
     end
+
+    @testset "show" begin
+        df = DataFrame(a=nothing, b=1)
+        io = IOBuffer()
+        show(io, DataFrameRow(df, 1, :))
+        @test String(take!(io)) == "DataFrameRow\n│ Row │ a       │ b     │\n│     │ Nothing │ $(Int) │\n" *
```

Comment:
`sprint` would be enough here, right? Also, a multiline string would be more readable.